src/features/search/components/VnSearchForm/VnSearchForm.tsx

```diff
@@ -0,0 +1,216 @@
+              <ControlWrapper label="Original Language">
+                <Field
+                  Component={LanguageSelect}
+                  control={control}
+                  name="originalLanguage"
+                  placeholder="Select languages"
+                  closeMenuOnSelect
+                  isClearable
+                />
+              </ControlWrapper>
+              <ControlWrapper label="Development status">
+                <Field
+                  Component={Select}
+                  control={control}
+                  name="developmentStatus"
+                  placeholder="Select development status"
+                  options={devStatusOptions}
+                  closeMenuOnSelect
+                  disableSearch
+                  isClearable
+                />
+              </ControlWrapper>
+              <ControlWrapper label="Length">
+                <Field
+                  Component={Select}
+                  control={control}
+                  name="length"
+                  placeholder="Select length"
+                  options={lengthOptions}
+                  closeMenuOnSelect
+                  disableSearch
+                  isClearable
+                />
+              </ControlWrapper>
+            </div>
+            <div className="grid grid-cols-2 gap-4">
+              <Field
+                Component={Slider}
+                control={control}
+                name="released"
+                label="Release Date"
+                showValues
+                min={1980}
+                max={2023}
+              />
+              <Field
+                Component={Slider}
+                name="rating"
+                control={control}
+                label="Rating"
+                showValues
+                min={10}
+                max={100}
+              />
+              <Field
+                Component={Slider}
+                name="popularity"
+                control={control}
+                label="Popularity"
+                showValues
+                min={0}
+                max={100}
+              />
+            </div>
+          </div>
+        </VnSearchPopover>
+
+      </div>
+
+      <div className="ml-auto flex gap-4">
+        <div className="flex">
+          <Controller
+            control={control}
+            name="sortDirection"
+            render={({ field: { onChange, value } }) => (
+              <IconButton
+                className="rounded-r-none"
+                intent="tertiary"
+                name={`sort-${value}`}
+                onClick={() => onChange(value === 'asc' ? 'desc' : 'asc')}
+              />
+            )}
+          />
+          <Field
+            Component={Select}
+            control={control}
+            options={sortFieldOptions}
+            closeMenuOnSelect
+            name="sortField"
+            className="rounded-r bg-gray-100"
+            disableSearch
+          />
```

Comment:
It seems that width of this select is based on length of the current label. Maybe we can improve it by making its width consistent regardless of label

Reply:
This will be fixed by @moetazaneta with updated design